tools/enso4igv/src/test/java/org/enso/tools/enso4igv/EnsoSbtProjectTest.java

```diff
@@ -0,0 +1,78 @@
+package org.enso.tools.enso4igv;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import org.netbeans.api.project.ProjectManager;
+import org.netbeans.api.project.Sources;
+import org.netbeans.junit.NbTestCase;
+import org.openide.filesystems.FileObject;
+import org.openide.filesystems.FileUtil;
+
+public class EnsoSbtProjectTest extends NbTestCase {
+
+  public EnsoSbtProjectTest(String name) {
+    super(name);
+  }
+
+  @Override
+  protected void setUp() throws Exception {
+    clearWorkDir();
+  }
+
+  public void testLanguageServerProject() throws Exception {
```

Comment:
Finally wrote a unit test. It emulates a structure of the `language-server` projects and checks how many "groups" are visible in the project view.